### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by syeopite**

### CODE REVIEW

**Comment by unixfox** [comment]
In testing on https://yewtu.be

File: shard.lock [hunk 1]
```diff
@@ -4,29 +4,33 @@ shards:
     git: https://github.com/athena-framework/negotiation.git
     version: 0.1.1
 
+  backtracer:
+    git: https://github.com/sija/backtracer.cr.git
+    version: 1.2.1
+
```
Comment (SamantazFox) [inline shard.lock:10]: What's this new one?

Comment (unixfox) [reply shard.lock:10]: > What's this new one?

https://github.com/crystal-loot/exception_page/commit/8f116768f3e2e5ad09dc212176c166d1de1468e9#diff-cbaa8062ee0fcfb5af8adfd98f69af4766b9ad5bb5e79158abe8da34f6fd7de0

**Comment by TheFrenchGhosty** [comment]
@unixfox 

> In testing on https://yewtu.be

Any news?

**Comment by unixfox** [comment]
> @unixfox
> 
> > In testing on https://yewtu.be
> 
> Any news?

I haven't seen any issue related to the upgrade of the dependencies.

**Pull request merged**